diff -dur corosync-2.1.0.orig/configure.ac corosync-2.1.0/configure.ac
--- corosync-2.1.0.orig/configure.ac	2012-10-10 10:17:50.000000000 +0200
+++ corosync-2.1.0/configure.ac	2012-10-25 08:59:49.000000000 +0200
@@ -611,8 +611,8 @@
 AC_SUBST([SONAME])
 
 AM_CONDITIONAL(INSTALL_TESTAGENTS, test -n "${enable_testagents}")
 AM_CONDITIONAL(INSTALL_MIB, test "${do_snmp}" = "1")
-AM_CONDITIONAL(INSTALL_DBUSCONF, test "${enable_dbus}" = "1")
+AM_CONDITIONAL(INSTALL_DBUSCONF, test "${enable_dbus}" = "yes")
 AM_CONDITIONAL(AUGTOOL, test -n "${AUGTOOL}")
 AC_SUBST([NSS_LDFLAGS])
 

