diff -dur corosync-2.1.0.orig/exec/Makefile.am corosync-2.1.0/exec/Makefile.am
--- corosync-2.1.0.orig/exec/Makefile.am	2012-10-10 10:17:50.000000000 +0200
+++ corosync-2.1.0/exec/Makefile.am	2012-10-25 08:49:37.000000000 +0200
@@ -50,7 +50,7 @@
 libtotem_pg_la_SOURCES	= $(TOTEM_SRC)
 libtotem_pg_la_CFLAGS	= $(nss_CFLAGS) $(rdmacm_CFLAGS) $(ibverbs_CFLAGS)
 libtotem_pg_la_LDFLAGS	= -version-number $(subst .,:,$(SONAME))
-libtotem_pg_la_LIBADD	= -lpthread $(nss_LIBS) $(rdmacm_LIBS) $(ibverbs_LIBS)
+libtotem_pg_la_LIBADD	= -lpthread $(LIBQB_LIBS) $(nss_LIBS) $(rdmacm_LIBS) $(ibverbs_LIBS)
 
 sbin_PROGRAMS		= corosync
 
Only in corosync-2.1.0/exec: Makefile.am~
