--- corosync-2.0.1.orig/configure.ac	2012-05-22 08:52:01.000000000 +0200
+++ corosync-2.0.1/configure.ac	2012-07-01 11:05:47.160318795 +0200
@@ -601,8 +601,8 @@
 
 AC_SUBST([OS_LDL])
 AM_CONDITIONAL(INSTALL_TESTAGENTS, test -n "${enable_testagents}")
 AM_CONDITIONAL(INSTALL_MIB, test "${enable_snmp}" = "1")
-AM_CONDITIONAL(INSTALL_DBUSCONF, test "${enable_dbus}" = "1")
+AM_CONDITIONAL(INSTALL_DBUSCONF, test "${enable_dbus}" = "yes")
 AM_CONDITIONAL(AUGTOOL, test -n "${AUGTOOL}")
 AC_SUBST([NSS_LDFLAGS])
 
--- corosync-2.0.1.orig/init/Makefile.am	2012-05-22 08:52:01.000000000 +0200
+++ corosync-2.0.1/init/Makefile.am	2012-07-01 11:11:22.512547818 +0200
@@ -39,10 +39,8 @@
 if INSTALL_SYSTEMD
 systemdconfdir	   = $(SYSTEMDDIR)
 systemdconf_DATA   = corosync.service corosync-notifyd.service
-initscriptdir	   = $(INITWRAPPERSDIR)
-else
-initscriptdir	   = $(INITDDIR)
 endif
+initscriptdir	   = $(INITDDIR)
 initscript_SCRIPTS  = corosync corosync-notifyd
 
 %: %.in Makefile
